ejercicios/ejercicio-010/ejercicio.py

```diff
@@ -5,7 +5,11 @@
 
 
 def tercera_letra(palabra):
-    pass
+
+   string = palabra
+   
+   print(string[2])
+   return string[2]
```

Comment:
Aprobado. Yo haria este cambio para que quede mas simple sin el print innecesario

```suggestion
   return palabra[2]
```